src/common/EmptyView/EmptyView.js

```diff
@@ -13,6 +13,7 @@ const EmptyView = ({
   docsUrl,
   description,
   urlLabel,
+  shouldBeHidden,
```

Comment:
ah now I got what you meant.   I deleted button label , the text went away. but the button stayed. that is why I added a new props.